### ISSUE TITLE
Enable user to write a custom code to modify a response

### ISSUE THREAD

**Issue by shubhendumadhukar** [fauxauldrich/camouflage#25]
**Is your feature request related to a problem? Please describe.**
Current {{#code}}{{/code}} helper, allows user to generate a complete response by writing a custom code which returns a response object. However there might be use cases where user doesn't want to write a code for whole response, instead user might just want to use custom code just for a part of the response. A new helper to return values by evaluating a javascript code, would be helpful

**Describe the solution you'd like**
Final implementation of this helper could look like `{{#inject}}(()=>{ return somevalue; })(){{/inject}}`

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by shubhendumadhukar** [comment]
In [src/handlebar/handleBarDefinition.ts](https://github.com/fauxauldrich/camouflage/blob/development/src/handlebar/handleBarDefinition.ts) file, create a new helper function:
```javascript
injectHelper = () => {
    Handlebars.registerHelper("inject", (context) => {
      const request: express.Request = context.data.root.request;
      const logger = context.data.root.logger;
      const result = eval(context.fn(this));
      return result;
    });
  };
```

Create a new file mocks/code/inject/GET.mock with following content:
```javascript
HTTP/1.1 200 OK
Content-Type: application/json

{
    "phone": {{#inject}} (()=>{ return Math.round(Math.random() * 10000000000); })();{{/inject}}
}
```

Build the application: `npm run build`
Run the application: `node bin/camouflage.js --config config.yaml`

Make a GET request to http://localhost:8080/code/inject, and validate the response.

Bonus points for writing in line documentation in comments and tests in index.spec.js

**Comment by shubhendumadhukar** [comment]
Implemented in #27
